core/src/main/java/org/apache/calcite/plan/RelOptUtil.java

```diff
@@ -286,6 +286,22 @@ public static Set<CorrelationId> getVariablesUsed(RelNode rel) {
     return visitor.vuv.variables;
   }
 
+  /**
+   * Returns a set of variables used by the given list of sub-queries and its descendants.
+   *
+   * <p> Internally it uses the {@link org.apache.calcite.plan.RelOptUtil#getVariablesUsed}
```

Comment:
In general I would refrain from describing *implementation* in JavaDoc, unless it is very relevant. JavaDoc is about what the function does, not how. This comment can be a regular comment inside the function, although the function is pretty clear as is.